Makefile.am

```diff
@@ -162,6 +162,11 @@ local-check:
 				TESTS="${current_tests} ${subdir_tests}"
 endif
 
+# We use VERSION.txt as VERSION file can cause C++ build errors on case-insensitive file-systems.
+# Some step(s?) still can produce or leave behind a VERSION file, cleanup it before packing the final dist tar.gz file
```

Comment:
"cleaning it up" would be the correct wording

Reply:
except, if you wish to say here something in "imperative mood", just like I wanted :)